src/functions-and-arrays.js

```diff
@@ -1,19 +1,67 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1, num2) {
+  if(num1 > num2){
+    return num1
+  }
+  else if(num2 > num1){
+    return num2
+  }
+  else{return num1}
+}
+
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
 
+function findLongestWord(arr) {
+  if(arr.length === 0) {
+    return null
+  }
+  else if(arr.length === 1) {
```

Comment:
No need to check for a length of 1, because your logic will work for a single word, it will return itself